core/core_bind.cpp

```diff
@@ -2188,7 +2188,27 @@ bool _ClassDB::is_class_enabled(StringName p_class) const {
 	return ClassDB::is_class_enabled(p_class);
 }
 
+Array _ClassDB::get_non_object_methods_list(Variant::Type p_type) const {
```

Comment:
I've had issues with using `Variant::Type` in a method signatures while generating C# mono glue and while generating/validating RST documentation ([see commit in `VariantResource` class in Goost](https://github.com/goostengine/goost/commit/6c93c9baf901110be081dc1031d5b9d53f33e30c)). Not sure if that's still an issue in `master`, though.

If that's still an issue, the workaround is to use `int` over `Variant::Type`.